6.4.remember_remember.py

```diff
@@ -0,0 +1,16 @@
+from PIL import Image

```

Comment:
This script doesn't work according to the task you received.
Fix it and resubmit.